qiita_db/user.py

```diff
@@ -224,10 +224,21 @@ def create(cls, email, password, info=None):
         # for sql insertion
         columns = info.keys()
         values = [info[col] for col in columns]
-
+        queue = "add_user_%s" % email
+        conn_handler.create_queue(queue)
+        # crete user
         sql = ("INSERT INTO qiita.%s (%s) VALUES (%s)" %
```

Comment:
I know you have not touch this code, but we have been (more or less) consistently using format for this type of changes. I would appreciate if you can change that. Non blocking...